test/runtest/external_tests.py

```diff
@@ -35,7 +35,7 @@
 import TestRuntest
 
 test = TestRuntest.TestRuntest()
-test.subdir('ext', ['ext', 'test'], ['ext', 'test', 'subdir'])
+test.subdir('ext/test/subdir')
```

Comment:
I thought you had to "declare" (create) all the test directories, since I've seen it done that way? This was a quick copy-and-mod of `test/runtest/testargv.py`, by the way. Ah, I see now the `subdir` docstring does talk about making intermediates, too.